developerportal/apps/people/models.py

```diff
@@ -263,18 +258,16 @@ def events(self):
         """
         from ..events.models import Event
 
-        upcoming_events = (
-            Event.objects.filter(start_date__gte=datetime.datetime.now())
-            .live()
-            .public()
+        upcoming_events = Event.published_objects.filter(
+            start_date__gte=datetime.datetime.now()
         )
 
-        speaker_events = Event.objects.none()
+        speaker_events = Event.published_objects.none()
```

Comment:
Tangent: what's the use-case for `none()` here?

Reply:
I believe the author’s intent here is to create an empty queryset and populate it with the subsequent loop. These methods can probably be refactored to be more efficient, I’ll create an issue.